bootstrap.py

```diff
@@ -639,6 +639,20 @@ def get_api_port():
 print "This script is designed to register new systems or to migrate an existing system to a Foreman server with Katello"
 
 
+def check_prerequisite():
+    rpm_sat = ['satellite', 'satellite-capsule']
```

Comment:
spacewalk-proxy-common & spacewalk-backend for Spacewalk/Sat5 proxy & Server respectively.